tests/integration_tests/test_sim.py

```diff
@@ -176,6 +176,18 @@ def test_temperature_read(qtbot, main_app):
 
         assert main_app.rocket_data.lastvalue(SubpacketEnum.TEMPERATURE.value) == vals[0]
 
+def test_time_update(qtbot, main_app: MainApp):
```

Comment:
```suggestion
def test_time_update(qtbot, main_app):
```

CI failing because MainApp is not imported. I wouldn't bother importing it just for this type hint.

Also after this is fixed. Is there anything else that needs to be done before we can merge both the PRs?

Reply:
Should be fixed now. Sorry there's been so many simple bugs―I wish I could actually see if my tests pass. Hopefully that's all fixed with your changes.

I've updated the commit hash for this commit on FLARE. I don't think anything else is required before merging.

Reply:
Nevermind seems like the Travis build is still failing. Any idea why?